configuration/wireless/access-point.md

```diff
@@ -27,35 +27,20 @@ sudo systemctl stop hostapd
 
 We are configuring a standalone network to act as a server, so the Raspberry Pi needs to have a static IP address assigned to the wireless port. This documentation assumes that we are using the standard 192.168.x.x IP addresses for our wireless network, so we will assign the server the IP address 192.168.0.1. It is also assumed that the wireless device being used is `wlan0`.
 
-First, the standard interface handling for `wlan0` needs to be disabled. Normally the dhcpcd daemon (DHCP client) will search the network for a DHCP server to assign a IP address to `wlan0`. This is disabled by editing the configuration file:
-
+To configure the static IP address, edit the dhcpcd configuration file with: 
 ```
 sudo nano /etc/dhcpcd.conf
 ```
-
-Add `denyinterfaces wlan0` to the end of the file (but above any other added `interface` lines) and save the file.
-
-To configure the static IP address, edit the interfaces configuration file with: 
-```
-sudo nano /etc/network/interfaces
-```
-Find the `wlan0` section and edit it so that it looks like the following:
-
-```
-allow-hotplug wlan0  
-iface wlan0 inet static  
-    address 192.168.0.1
-    netmask 255.255.255.0
-    network 192.168.0.0
+Go to the end of the file and edit it so that it looks like the following:
 ```
+interface wlan0
+    static ip_address=192.168.4.1/24
 
 Now restart the dhcpcd daemon and set up the new `wlan0` configuration:
 
 ```
 sudo service dhcpcd restart
-sudo ifdown wlan0
-sudo ifup wlan0
-```
+``
 
```

Comment:
minor typo here -  

"Now restart the dhcpcd daemon and set up the new `wlan0` configuration:" is inside the code block, 
"sudo service dhcpcd restart" is not